synapse/cortex.py

```diff
@@ -4710,6 +4715,23 @@ async def _onAddCronJob(self, cdef):
 
         return cdef
 
+    async def moveCronJob(self, useriden, croniden, viewiden):
+        view = self._viewFromOpts({'view': viewiden, 'user': useriden})
+
+        appt = self.agenda.appts.get(croniden)
+        if appt is None:
+            raise s_exc.NoSuchIden(iden=croniden)
+
+        if appt.view == view.iden:
+            return view.iden
+
```

Comment:
This is returning `view.iden` but onMoveCronJob returns the cron iden, seems like maybe we should be returning the cron iden here as well?

Reply:
Aye. I'll change that.